ui/app/templates/components/get-credentials-card.hbs

```diff
@@ -6,24 +6,32 @@
     <p class="is-label search-label">{{@searchLabel}}</p>
   </div>
   <p class="sub-text">{{@subText}}</p>
-  <SearchSelect
-    @id={{this.id}}
-    @models={{@models}}
-    @selectLimit="1"
-    @backend={{@backend}}
-    @fallbackComponent="input-search"
-    @shouldUseFallback={{@shouldUseFallback}}
-    @onChange={{action "handleRoleInput"}}
-    @inputValue={{get this.model this.valuePath}}
-    @placeHolder={{@placeHolder}}
-    data-test-search-roles
-  />
-  <input
+  {{#if @renderInputSearch}}
+    <InputSearch
+      @id="search-input-{{@type}}"
+      @onChange={{this.handleInput}}
+      @placeholder={{@placeholder}}
+      data-test-search-roles
+    />
+  {{else}}
+    <SearchSelect
+      @id="search-input-{{@type}}"
+      @models={{@models}}
+      @selectLimit="1"
+      @backend={{@backend}}
+      @fallbackComponent="input-search"
+      @onChange={{this.handleInput}}
+      @placeholder={{@placeholder}}
+      data-test-search-roles
+    />
+  {{/if}}
+  <button
     type="submit"
-    value={{@title}}
     class="button is-secondary"
     disabled={{this.buttonDisabled}}
-    onclick={{action "transitionToCredential"}}
```

Comment:
While I was in here cleaned this up - but if it feels outside the scope of this PR can remove!

Reply:
Thank you for the cleanup. The only reason to remove to another PR is if you feel this would need to be backported? If not, I'm fine keeping it in.

Reply:
I don't feel like it needs to be backported? That's a good point about button vs input. I'll confirm with design!

Reply:
The functionality is essentially the same so I don't think it needs to be backported